cmd/internal/cli/actions.go

```diff
@@ -59,26 +60,31 @@ func handleOCI(cmd *cobra.Command, u string) (string, error) {
 		return "", fmt.Errorf("failed to get SHA of %v: %v", u, err)
 	}
 
+	imgabs := ""
 	name := uri.GetName(u)
-	imgabs := cache.OciTempImage(sum, name)
 
-	if exists, err := cache.OciTempExists(sum, name); err != nil {
-		return "", fmt.Errorf("unable to check if %v exists: %v", imgabs, err)
-	} else if !exists {
-		sylog.Infof("Converting OCI blobs to SIF format")
+	if DisableCacheActions {
+		file, err := ioutil.TempDir(tmpDir, "single_use_cache_")
```

Comment:
We can change `file` to `imgabs` and avoid the assignment 5 lines below.

also why are you using `TempDir()` instead of `TempFile()`? If you meant to use `TempFile()` then the assignment below is not avoidable.

Reply:
> also why are you using `TempDir()` instead of `TempFile()`? If you meant to use `TempFile()` then the assignment below is not avoidable.

Because this function pulls a docker image, and that uses a tmp directory to store the layers, afaik (could be wrong)